test/linalg.jl

```diff
@@ -0,0 +1,64 @@
+using NiSparseArrays:imul!
+using SparseArrays, Random, Test
+Random.seed!(1234)
+
+# Is declaring approx_rtol here safe? But I don't want to declare approx_rtol in each testset...
+approx_rtol = 100*eps()
+# Would it be reasonable to test accuracy like this? Is there any better readable code?
```

Comment:
Looks good to me. Maybe using `const` is better, it will make the code type stable.

```julia
const approx_rtol = 100*eps()
```

Reply:
Thanks and it has been corrected~